mangaki/mangaki/tests/test_reco.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
```

Comment:
When possible, avoid supposing that `django.contrib.auth.models.User` is the default User model, rather use `django.contrib.auth.get_user_model`.